### ISSUE TITLE
Broken Test due to cassava updates

### ISSUE THREAD

**Issue by aycanirican** [jb55/pipes-csv#12]
One of the tests is failing with below output:
```
Running 1 test suites...
Test suite hunit: RUNNING...
Pipes.Csv.:
  HUnit: [OK]
  HUnit: [Failed]
expected: [Right (1,2),Right (1,2),Right (2,3),Left "expected Int, got \"one\" (Failed reading: takeWhile1)",Left "cannot unpack array of length 1 into a pair. Input record: fromList [\"no line end needed on last chunk\"]",Right (0,0)]
 but got: [Right (1,2),Right (1,2),Right (2,3),Left "expected Int, got \"one\" (Failed reading: takeWhile1)",Left "cannot unpack array of length 1 into a pair. Input record: [\"no line end needed on last chunk\"]",Right (0,0)]
  HUnit: [OK]
  HUnit: [OK]

         Test Cases  Total      
 Passed  3           3          
 Failed  1           1          
 Total   4           4          
Test suite hunit: FAIL
Test suite logged to: dist/test/pipes-csv-1.4.0-hunit.log
0 of 1 test suites (0 of 1 test cases) passed.
builder for ‘/nix/store/gak2i6wjs9sirmmqyga0r2ipr6k5lfkn-haskell-pipes-csv-1.4.0.drv’ failed with exit code 1
```

This is probably due to change in latest cassava (0.11.0.0):

```
-showsPrec p v = showParen (p > 10) $ showString "fromList " . shows (toList v)
+showsPrec _ = shows . toList
```

Do you mind fixing it?

**Comment by jb55** [comment]
Sure I'll look into this today

**Comment by jb55** [comment]
Fixed in 1.4.1: https://hackage.haskell.org/package/pipes-csv-1.4.1
